test/test_stack/test_topk.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import redis
+
+
+def test_topk_incrby(r: redis.Redis):
+    assert r.topk().reserve("topk", 3, 10, 3, 1)
+    assert [None, None, None] == r.topk().incrby(
+        "topk", ["bar", "baz", "42"], [3, 6, 2]
+    )
+    assert [None, "bar"] == r.topk().incrby("topk", ["42", "xyzzy"], [8, 4])
+    with pytest.deprecated_call():
+        assert [3, 6, 10, 4, 0] == r.topk().count(
+            "topk", "bar", "baz", "42", "xyzzy", 4
```

Comment:
@cunla Is this a stable test? I got a random failure https://github.com/alessio-locatelli/fakeredis-py/actions/runs/12491347330/job/34857256703?pr=1 but locally with Python 3.12 I cannot reproduce it.

Reply:
Yeah, sometimes this test fails. Would you like to help fixing it?

Reply:
To be honest, I've never used `topk`, but I'll have a look at it in the meantime if the test isn't stabilized sooner.